h2o-core/src/main/java/water/init/NetworkInit.java

```diff
@@ -185,6 +197,59 @@ public static void initializeNetworkSockets( ) {
     H2O.CLOUD_MULTICAST_PORT = NetworkUtils.getMulticastPort(hash);
   }
 
+  public static H2OHttpConfig webServerParams(H2O.OptArgs args) {
+    final H2OHttpConfig params = new H2OHttpConfig();
```

Comment:
Is there any way to avoid the copying of parameters here? This could be prone to errors.

At the minimum, I would suggest adding a test.